src/plugins/files/server/blob_storage_service/adapters/es/content_stream/content_stream.ts

```diff
@@ -110,10 +139,22 @@ export class ContentStream extends Duplex {
         chunks.push(chunk);
       }
       const buffer = Buffer.concat(chunks);
-      const source: undefined | FileChunkDocument = buffer.byteLength
-        ? cborx.decode(Buffer.concat(chunks))?._source
+      const decodedChunkDoc: GetResponse<FileChunkDocument> | undefined = buffer.byteLength
+        ? (cborx.decode(buffer) as GetResponse<FileChunkDocument>)
```

Comment:
FYI. I had a merge conflict here due to a change that was done in Main, but never backported. The change is in 8.7 now which I think is ok

diff: 

<img width="1616" alt="image" src="https://user-images.githubusercontent.com/56442535/226743392-ec707c1e-cda7-4ad2-af63-013eb74612d8.png">